templates/docs/examples/layouts/documentation-full-width.html

```diff
@@ -0,0 +1,422 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Documentation / Full width{% endblock %}
+
+{% block style %}
+<style>
+  body { margin: 0; }
+</style>
+{% endblock %}
+
+{% set is_paper = True %}
+{% block content %}
+
+
+<div class="l-docs">
+  <div class="l-docs__header">
+    <header id="navigation" class="p-navigation is-dark">
+      <div class="l-docs__container">
+        <div class="l-docs__sidebar">
+          <div class="p-navigation__banner">
+            <div class="p-navigation__tagged-logo">
+              <a class="p-navigation__link" href="#">
+                <div class="p-navigation__logo-tag">
+                  <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
+                </div>
+                <span class="p-navigation__logo-title">Ubuntu</span>
+              </a>
+            </div>
+            <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+            <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+          </div>
+        </div>
+        <div class="l-docs__main">
+          <div class="p-navigation__row u-fixed-width">
+            <nav class="p-navigation__nav" aria-label="Example main">
+              <ul class="p-navigation__items">
+                <li class="p-navigation__item">
+                  <a class="p-navigation__link" href="#">Products</a>
+                </li>
+                <li class="p-navigation__item">
+                  <a class="p-navigation__link" href="#">Services</a>
+                </li>
+                <li class="p-navigation__item">
+                  <a class="p-navigation__link" href="#">Partners</a>
+                </li>
+                <li class="p-navigation__item is-selected">
+                  <a class="p-navigation__link" href="#">Docs</a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section id="search-docs" class="p-strip is-bordered is-shallow l-docs__container">
+      <div class="l-docs__sidebar u-hide--medium u-hide--small">
+        <div class="u-fixed-width">
+          <h5 class="u-sv-2">Documentation example</h5>
+        </div>
+      </div>
+      <div class="l-docs__main">
+        <div class="row">
+          <form class="p-search-box u-no-margin--bottom">
+            <input type="search" class="p-search-box__input" name="q" placeholder="Search documentation" required="" autocomplete="on"/>
+            <button type="reset" class="p-search-box__reset" name="close"><i class="p-icon--close">Close</i></button>
+            <button type="submit" class="p-search-box__button" name="submitSearch"><i class="p-icon--search">Search</i></button>
+          </form>
+        </div>
+        <!--div class="row grid-demo">
```

Comment:
is this commented part really needed? if so, could we add a comment explaining why do we still want to keep it?

Reply:
No, it's leftover from testing. Thanks.